assets/js/app.js

```diff
@@ -34,6 +38,7 @@ function connect() {
           // Pass the most recent user data to the LiveView in `connect_params`
           user_data: loadUserData(),
           app_auth_token: loadAppAuthToken(),
+          confirm_opt_out_ids: loadConfirmOptOutIds(),
```

Comment:
FTR I think it's fine to send this on every connect. This list is just a few items at most, and now that we don't send payload for every confirm-decorated action we definitely send less data in total.

Reply:
I wonder if instead we could keep this fully on the client by adding a `data-confirm-id` on the server. Then, if already opted-out, we send an information on click saying "this has been opted-out"?

Reply:
This would work in some cases, but the confirm could totally be triggered in `handle_info` :D Also attaching the confirm id to each event and making sure it's the same sounds error-prone.

Reply:
Oh, I see. It can be activated from any time, we don't pre-render anything on the page. Sounds good to me then, ship it!